private/buf/bufgen/internal/bufgenv2/config.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 // disableFunc decides whether a file option should be disabled for a file.
-type disabledFunc func(fileOption, imageFileIdentity) bool
+type disabledFunc func(FileOption, imageFileIdentity) bool
```

Comment:
Occurrences of `fileOption` -> `FileOption` are the only changes in this file

Reply:
Optimally, this shouldn't be public either.